services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDocumentBuilder.java

```diff
@@ -51,16 +51,16 @@ public MongoDbDocumentBuilder withDeviceId(final String deviceId) {
     /**
      * Adds nested CA subjectDn property.
```

Comment:
How about adding a description in sync with the other methods in this class? Something like `Sets the json object with the given subject DN.`

Reply:
done